code/__DEFINES/preferences.dm

```diff
@@ -19,12 +19,13 @@
 #define CHAT_DEBUGLOGS	128
 #define CHAT_LOOC		256
 #define CHAT_GHOSTRADIO 512
-#define SHOW_TYPING 	1024
-#define DISABLE_SCOREBOARD 2048
-#define DISABLE_KARMA_REMINDER	4096
-#define MEMBER_PUBLIC	8192
-#define CHAT_NO_ADMINLOGS 16384
-#define DONATOR_PUBLIC	32768
+#define CHAT_GHOSTANONSAY 1024
+#define SHOW_TYPING 	2048
+#define DISABLE_SCOREBOARD 4096
+#define DISABLE_KARMA_REMINDER	8192
+#define MEMBER_PUBLIC	16384
+#define CHAT_NO_ADMINLOGS 32768
+#define DONATOR_PUBLIC	65536
```

Comment:
BYOND's bit operators don't let you go above 32768, so this won't work. And if it did, you can't shift the existing ones down like this, because that jumbles the bits in existing preferences.